tincr/io/library/genCellLibrary.tcl

```diff
@@ -822,7 +822,9 @@ proc ::tincr::write_macro_xml {macro outfile} {
     puts $outfile "        <cells>"
     foreach internal [get_cells $macro/*] {
         puts $outfile "            <internal>"
-        puts $outfile "                <name>[lindex [split [get_property NAME $internal] "/"] 1]</name>"
+        set cell_name [get_property NAME $internal]
+        set first [expr {[string first "/" $cell_name] + 1}]
```

Comment:
Seems like overkill to me. Also, wouldn't it be slower? The parsing here is pretty simple.